requirements/pip_tools.txt

```diff
@@ -4,7 +4,7 @@
 #
 #    make upgrade
 #
-build==0.8.0
+build==0.9.0
```

Comment:
Dropped support for `Python 3.6`: https://pypa-build.readthedocs.io/en/stable/changelog.html#id1